Foundation/CPKeyValueObserving.j

```diff
@@ -1300,6 +1299,10 @@ var kvoNewAndOld        = CPKeyValueObservingOptionNew | CPKeyValueObservingOpti
     if (aKeyPath === _firstPart)
```

Comment:
The only change is that this part is inlined one level except for the first if statement.
Before this pull request the first if statement just returned from this method if the key was already replace. Now it will only skip the first part in the method.